components/mdc/TextInput/index.js

```diff
@@ -4,3 +4,7 @@ import TextField from './TextField.svelte'
 import MoneyInput from './MoneyInput.svelte'
 
 export { TextArea, TextField, MoneyInput }
+
+export const addOrRemoveInvalidClass = async (isInvalid, element) => {
+  isInvalid ? element.classList.add('mdc-text-field--invalid') : element.classList.remove('mdc-text-field--invalid')
```

Comment:
Is this also the class to use for test _areas_ that are invalid?

Reply:
yes, they share the same README so I this applies to both "To change the invalid state of your text fields, apply these mixins with CSS selectors such as .foo-text-field.mdc-text-field--invalid."